.github/workflows/ci.yml

```diff
@@ -150,10 +150,10 @@ jobs:
             ~/.cargo/git/db/
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/rust-toolchain.toml') }}-${{ hashFiles('**/Cargo.lock') }}
-      # todo!(windows): Actually run clippy
-      #- name: cargo clippy
-      #  shell: bash -euxo pipefail {0}
-      #  run: script/clippy
+
+      - name: cargo clippy
+        shell: bash -euxo pipefail {0}
```

Comment:
Passing by and leaving a note, feel free to ignore for now: this is super odd for Windows, isn't it?

`shell: pwsh` looks much more appropriate

Reply:
I considered doing that, but it would require a new `pwsh` script like `clippy.ps1`, which would increase maintenance costs. I left it as it is, since Windows runner has several instances of Bash installed, and there seems to be no problem with scripts that simply execute cargo commands.

Reply:
What I did notice when running the clippy command on my Win machine locally via PS is that there were more failures, is that the case for you?
Never mind me if it's not.

Reply:
I tried running the commands in `script/clippy` directly from Powershell with no errors. If I run `cargo clippy` without options, it fails, but I think that is the same in all environments.

Reply:
Ok, thank you for double checking.
I will try it again on my machine and come back with PRs if I find something odd.